src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -107,10 +108,10 @@ private Future<CompositePurchaseOrder> updateAndGetOrderWithLines(CompositePurch
   }
 
   private void makePoLinePending(CompositePoLine poLine) {
-    if (poLine.getPaymentStatus() == CompositePoLine.PaymentStatus.AWAITING_PAYMENT) {
+    if (EnumSet.of(CompositePoLine.PaymentStatus.AWAITING_PAYMENT, CompositePoLine.PaymentStatus.ONGOING).contains(poLine.getPaymentStatus())) {
```

Comment:
is it possible to write some test here that both conditions are covered?

Reply:
> is it possible to write some test here that both conditions are covered?

I'll add a Karate test next week to cover the whole Unopening scenario